lib/s3Create.js

```diff
@@ -13,7 +13,7 @@ const s3 = new AWS.S3({apiVersion: '2006-03-01'})
 const s3Create = function (file, mimetype) {
   const fileStream = fs.createReadStream(file)
   fileStream.on('error', function (err) {
-    console.log('File Error', err)
+    return ('File Error', err)
```

Comment:
I'm not sure if you can do a return with multiple parameters like this. I'd just return the error, we have error handlers later to handle it.

Reply:
sounds good I will change